turbopack/crates/turbopack-ecmascript-runtime/js/src/browser/dev/runtime/dom/runtime-backend-dom.ts

```diff
@@ -236,53 +236,61 @@ async function loadWebAssemblyModule(
     const chunkUrl = getChunkRelativeUrl(chunkPath);
     const decodedChunkUrl = decodeURI(chunkUrl);
 
-    if (chunkPath.endsWith(".css")) {
-      const previousLinks = document.querySelectorAll(
-        `link[rel=stylesheet][href="${chunkUrl}"],link[rel=stylesheet][href^="${chunkUrl}?"],link[rel=stylesheet][href="${decodedChunkUrl}"],link[rel=stylesheet][href^="${decodedChunkUrl}?"]`
-      );
-      if (previousLinks.length > 0) {
-        // CSS chunks do not register themselves, and as such must be marked as
-        // loaded instantly.
-        resolver.resolve();
+    if (typeof importScripts === "function") {
+      if (chunkPath.endsWith(".js")) {
```

Comment:
What should we do about css in WebWorkers? Should we error or silently ignore that?

Reply:
I've ignored them now